.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env sh
+
+TARGETARCH=amd64
+
+# Install Kind
+if ! command -v kubectl &> /dev/null; then
```

Comment:
@csantanapr - should it be `command -v kind` instead of `command -v kubectl` at this point?